lib/Rfc1035StubResolver.php

```diff
@@ -88,8 +94,25 @@ public function resolve(string $name, int $typeRestriction = null): Promise
         }
 
         return call(function () use ($name, $typeRestriction) {
-            if (!$this->config) {
-                yield $this->reloadConfig();
+            if (!$this->configStatus) {
```

Comment:
Please use `$this->configStatus === self::CONFIG_NOT_LOADED`

Reply:
done